tools/run_tests/run_interop_tests.py

```diff
@@ -589,6 +602,11 @@ def aggregate_http2_results(stdout):
                   action='store_const',
                   const=True,
                   help='Run cloud_to_prod_auth tests.')
+argp.add_argument('-p', '--prod_servers',
+                  default=prod_servers,
+                  action='store_const',
+                  const=True,
+                  help='The server to run cloud_to_prod_auth tests against.')
```

Comment:
nit: not just cloud_to_prod_auth tests, both cloud_to_prod and cloud_to_prod_auth (if requested by corresponding cmdline arguments of course).